app/views/admin_mailer/abuse_report.html.erb

```diff
@@ -4,6 +4,11 @@
   
   Email of the person who reported the abuse, if given: <%= style_email(@email, @email) %><br>
   URL of the page the abuse is on: <%= style_link(@url, @url) %><br>
+  
+  <% if @has_metadata %>
+    Title of the work: <%= @title.html_safe %><br/> 
+    Author(s) of the work: <%= @author.html_safe %><br/> 
```

Comment:
Whoops, looks like my last comment got eaten by updates. The emails should say "Creator(s)" rather than "Author(s)" because we're trying to move away from text-centric language.